temporalio/worker/worker.py

```diff
@@ -71,6 +71,7 @@ def __init__(
         graceful_shutdown_timeout: timedelta = timedelta(),
         shared_state_manager: Optional[SharedStateManager] = None,
         debug_mode: bool = False,
+        disable_eager_activity_execution: bool = False,
```

Comment:
I'd rather maintain consistency between the SDKs.
Looks like in Go we expose both `MaxConcurrentEagerActivityExecutionSize` and a disable flag.
In Python we now only have a disable flag.
In Java / TS we do not expose eager activity worker options.

I don't think we should expose this setting for now in Python.
Later on we might want to expose `max_eager_activity_requests_per_task` which is hardcoded to 3 for now.
Let's wait and do some more testing so we can give out advice how to tune this parameter.

BTW, while reviewing the Go worker options, it looks like we either have the wrong behavior or the `MaxConcurrentEagerActivityExecutionSize` doesn't do what it says it does, I was lazy didn't bother to read the implementation. I missed this in my review of your Go PR that set the default to 3.

Reply:
Can you clarify exactly what I should put wrt eager activities settings today? We of course want to make sure people can disable it if broken (something we didn't have in last beta).

I will check the Go implementation. Can you open an issue or put somewhere how you're testing that it's not working?

Reply:
https://github.com/temporalio/sdk-go/issues/950

Reply:
Thanks for the Go issue, will investigate.

Can you clarify exactly what I should put wrt eager activities settings today? As in this PR before Python beta release?

Reply:
Let's keep the disable flag but mark it experimental.
Later on we'll allow max per task (0 == disable) and, pending discussion, consider adding max concurrent eager.